Deeploy/Targets/GAP9/Platform.py

```
@@ -0,0 +1,306 @@
+        super().__init__(memoryHierarchy, defaultTargetMemoryLevel, engines, variableBuffer, constantBuffer,
+                         structBuffer, transientBuffer)
+
+    def getTargetMemoryLevel(self, node: gs.Node, tensorName: str, ctxt: NetworkContext) -> str:
+        if node.op in self.untiledOps:
+            return ctxt.lookup(tensorName)._memoryLevel
+        return super().getTargetMemoryLevel(node, tensorName, ctxt)
+
+
+class MemoryGAP9PlatformWrapper(MemoryPlatformWrapper):
+
+    untiledOps = []
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)